storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/schema/V3Schema.java

```diff
@@ -42,6 +46,12 @@
       RocksDbColumn.create(5, CHECKPOINT_SERIALIZER, STATE_SERIALIZER);
   RocksDbColumn<UnsignedLong, VoteTracker> VOTES =
       RocksDbColumn.create(6, UNSIGNED_LONG_SERIALIZER, VOTES_SERIALIZER);
+  RocksDbColumn<UnsignedLong, Bytes32> ETH1_DEPOSIT_HASHES =
+      RocksDbColumn.create(7, UNSIGNED_LONG_SERIALIZER, BYTES32_SERIALIZER);
+  RocksDbColumn<UnsignedLong, DepositData> ETH1_DEPOSIT_CACHE =
+      RocksDbColumn.create(8, UNSIGNED_LONG_SERIALIZER, DEPOSIT_DATA_SERIALIZER);
+  RocksDbColumn<UnsignedLong, Eth1BlockData> ETH1_BLOCK_METADATA =
+      RocksDbColumn.create(9, UNSIGNED_LONG_SERIALIZER, ETH1_BLOCK_DATA_SERIALIZER);
```

Comment:
Once we commit this we need to be sure we can maintain backwards compatibility as we won't be able to drop these columns again without migrating data to another db format (and with users syncing testnets now backwards compatibility and migration between versions is going to be important).  We can delete any existing data in these fields and just reload from ETH1 though, but need to have a plan of how we're going to work out if we need to discard the data or not.